iron-form.html

```diff
@@ -137,13 +147,18 @@
 
       this._requestBot.generateRequest();
       this.fire('iron-form-submit', json);
+
+      if (this.allowActionRedirect) {
+        var redirectUrl = this.action + '?' + this._requestBot.queryString;
+        window.location = redirectUrl;
```

Comment:
as `redirectUrl` is only used to update `window.location`, is it worth avoiding the variable allocation by just assigning to `window.location` directly? i.e ```window.location = this.action + '?' + this._requestBot.queryString;```